src/main/java/com/sfuentes/Theater.java

```diff
@@ -0,0 +1,30 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+
+public class Theater {
+
+  ArrayList<Integer> ages;
+  final double price = 20d;
+
+  public Theater(ArrayList<Integer> age) {
+    this.ages = age;
+  }
+
+  public double getSubtractionByDiscount() {
```

Comment:
I think the problem  ask the discount by category not the total discount

Reply:
Fixe